airqo_monitor/utils.py

```
@@ -9,6 +9,21 @@
 )
 
 
+def get_int_global_var_value(key):
```

do we need any error checking or anything when we get globalvariables?

---

i think we need to let it crash because not getting the globalvar correctly will break harder to catch things down the line